Packs/Hey/Scripts/Hey/Hey.py

```diff
@@ -0,0 +1,199 @@
+import re
+import subprocess
+from typing import Tuple
+
+import urllib3
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+# ---------- CONSTANTS ---------- #
+
+FLOAT_RE = r'\d+\.\d+'
+INT_RE = r'\d+'
+BYTES_RE = r'\d+ bytes'
+
+
+# ---------- HELPER FUNCTIONS ---------- #
+
+
+def try_re(pattern: str, string: str, i: int = 0) -> Optional[Any]:
+    re_res = re.findall(pattern, string)
+    if len(re_res) == i + 1:  # ignore cases where len is greater than expected
+        return re_res[i]
+    return None
+
+
+def semicolon_arg_to_dict(arg: Optional[str]):
```

Comment:
Not sure why this is named `semicolon`. If I understand correctly you are splitting on `name=value`. Maybe this method should be named: `name_value_arg_to_dict` to make things clearer.

Reply:
missed that one - I change it from a semicolon to a comma. In any case, I'll take your advice.